string name
string parent_link_name
geometry_msgs/Pose local_pose

float64 time
geometry_msgs/Pose pose
float64[] wrench

float64[] M
float64[] D
float64[] K

float64[] act_wrench
float64[] wrench_gain

bool support_com
